.ci_support/migrations/cuda12x.yaml

```diff
@@ -0,0 +1,36 @@
+migrator_ts: 1738229377
+__migrator:
+  kind:
+    version
+  # Use migrator to add CUDA 12 latest.
+  use_local: true
```

Comment:
Any particular reason why you're using a custom migrator (`cuda12x.yaml`) as opposed to the one from the global pinning (`cuda129.yaml`)?

Reply:
There's a long history of using a custom migrator like that here and elsewhere to ease the update process, which is baked into my muscle memory. Often because we have wanted to update feedstocks pretty quickly (including for CUDA minor versions). Also due to occasional quirks with particular (like compiler versions they support)

That said, agree it is probably less relevant in this case given there is a global migrator. Plus we've reached the end of the CUDA 12 line

Switching to the global one: https://github.com/conda-forge/xgboost-feedstock/pull/252